code/API_definitions/qos-profiles.yaml

```
@@ -339,6 +339,48 @@ components:
           minimum: 1
           maximum: 10
           example: 3
+        l4sQueueType:
+          type: string
+          enum:
+            - non-l4s-queue
+            - l4s-queue
+            - mixed-queue
+          description: Specifies the type of queue for L4S traffic management
+        differentiatedServicesCodepoint:
+          type: string
+          description: |
+            The name of the service class requested by the API client. It is associated with QoS behaviors optimized for a particular application type.
```

The QoS Profile is the description of the quality of service offered by the network when the profile is activated, it is not "requested" by the API Client (the API client can only request the QoS Profile as described).

Some suggestion, but need to be improved:
```suggestion
            The name of a Service Class, indicating that the QoS Profile is offering an optimized behavior for a particular application type. DSCP might or might not be used within the network when the QoS profile is used in a QoS Session (it could be used also only on certain segments of the e2e connection).
```

---

That one was resolved without comment?

---

oops, not sure what happened @hdamker - thanks for the feedback. I updated the description - take a look and let me know if you have any additional feedback.